policy/auxTelMapper.yaml

```diff
@@ -0,0 +1,411 @@
+defects: ../description/defects
+
+needCalibRegistry: true
+
+levels:
+  tract: patch
+
+defaultLevel: visit
+
+exposures:
+  _raw:
+    level: Ccd
+    persistable: PropertyList
+    python: lsst.daf.base.PropertyList
+    storage: FitsStorage
+    tables: raw
+    template: raw/%(ccd)s/%(ccd)s-%(visit)06d.fits[0]
+  raw:
+    assembler: lsst.obs.auxTel.auxTelMapper.assemble_raw
+    composite:
+      raw_amp:
+        datasetType: raw_amp
+        subset: true
+      raw_hdu:
+        datasetType: raw_hdu
+    inputOnly: true
+  raw_amp:
+    level: Amp
+    # NB If type is changed to an exposureI then constructDark breaks. If changing this be sure to test calibs
+    persistable: DecoratedImageI
+    python: lsst.afw.image.DecoratedImageI
+    storage: FitsStorage
+    tables: raw
+    template: raw/%(ccd)s/%(ccd)s-%(visit)06d.fits[%(channel)d]
+  postISRCCD:
+    template: 'postISRCCD/postISRCCD_v%(visit)d_%(ccd)s_f%(filter)s.fits'
```

Comment:
This is the only template that is quoted.  I think it would be nice to be consistent and remove the quotes if this survives the culling of duplicated material.

Reply:
I have no idea why it was like this, nor why the others weren't. They are all the same now.